examples/helloworld_helm/helloworld_helm_agent.go

```diff
@@ -0,0 +1,157 @@
+package main
+
```

Comment:
can we just build a common helloworld agent? It seems each example shows the same agent.